js/main.js

```diff
@@ -1,26 +1,128 @@
+  'Милое бунгало',
+  'Дворец для царей',
+  'Отель хостел',
+];
 
-  return NaN;
-};
+const TYPES = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel',
+];
+
+const CHECKIN = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const CHECKOUT = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const FEATURES = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner',
+];
 
-getRandomInteger(33, 45);
+const DESCRIPTION = [
+  'Останавливайтесь здесь и не пожалеете',
+  'Лучшее место, чтобы придаться блаженству',
+  'Виды - это наш конек! ',
+  'Вдали от шумных улиц и запахов еды',
+  'Самое топовое место среди аутентичных улиц и кучи проводов',
+];
 
-// функция возврата положительного числа с плавающей точкой
-const getRandomFloat = (min, max, round) => {
-  if (min < 0 || max < 0 || max < min) {
+const PHOTOS = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const SIMILAR_ADS = 10;
+
+const getRandomPositiveInteger = (a, b) => {
+  if (a < 0 || b < 0) {
+    return NaN;
+  }
+  const lower = Math.ceil(Math.min(a, b));
+  const upper = Math.floor(Math.max(a, b));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
+};
 
+function getRandomPositiveFloat (a, b, digits = 5) {
+  if (a < 0 || b < 0 || digits < 0) {
     return NaN;
   }
-  const floatNumber = ((Math.random() * (max - min + 1)) + min).toFixed(round);
+  const lower = Math.min(a, b);
+  const upper = Math.max(a, b);
+  const result = Math.random() * (upper - lower) + lower;
+  return +result.toFixed(digits);
+}
 
-  return Number(floatNumber);
+function getArray(array) {
+  const maxLength = array.length;
+  const lengthOfArray = getRandomPositiveInteger(1, maxLength);
+  const arrayRandom = [];
+
+  while (arrayRandom.length < lengthOfArray) {
+    const indexOfEl = getRandomPositiveInteger(0, maxLength - 1);
+    const el = array[indexOfEl];
+
+    if (!arrayRandom.includes(el)) {
+      arrayRandom.push(el);
+    }
+  }
+  return arrayRandom;
+}
+
+const adsNearby = () => {
+  const getRandomArrayElement = (elements) => elements[getRandomPositiveInteger(0, elements.length - 1)];
+  let randomPhotoIndex = getRandomPositiveInteger(1, 10);
+  const randomPrice = getRandomPositiveInteger (0, 10000000);
+  const randomRooms = getRandomPositiveInteger (0, 1000);
+  const randomGuests = getRandomPositiveInteger (0, 1000);
+  const randomLat = getRandomPositiveFloat (35.65000, 35.70000);
+  const randomLng = getRandomPositiveFloat (139.70000, 139.80000);
+
+  if (randomPhotoIndex < 10) {
+    randomPhotoIndex = `0${ randomPhotoIndex}`;
+  }
+
+  return {
+    author: `img/avatars/user${ randomPhotoIndex }.png`,
```

Comment:
По условиям дз в объекте author должно быть поле avatar в которое мы и записываем эту строку
```suggestion
    author: `img/avatars/user${randomPhotoIndex}.png`,
```